content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+---
+title: "Contact Routing Guide"
+linkTitle: "Contact Routing Guide"
+description: "Set up a Sensu Enterprise Contact Routing."
+product: "Sensu Enterprise"
+version: "2.6"
+weight: 7
+menu:
+ sensu-enterprise-2.6:
+   parent: guides
+---
+
+- [Prerequisites](#prerequisites)
+- [Overview](#overview)
+  - [Contact Routing Basics](#contact-routing-basics)
+    - [Contact Configuration](#contact-configuration)
+    - [Check Configuration](#check-configuration)
+    - [Client Configuration](#client-configuration)
+- [Example Implementation](#example-implementation)
+  - [Default Configuration](#default-configuration)
+  - [Single Handler](#single-handler)
+      - [Single Handler with a Single Matching Contact](#single-handler-with-a-single-matching-contact)
+      - [Single Handler with a Single Non-Matching Contact](#single-handler-a-single-non-matching-contact)
+      - [Single Handler with Multiple Matching Contacts](#single-handler-with-multiple-matching-contacts)
+      - [Single Handler with Multiple Non-Matching Contacts](#single-handler-with-multiple-non-matching-contacts)
+      - [Single Handler with Some Matching Contacts](#single-handler-with-some-matching-contacts)
+  - [Multiple Handlers](#multiple-handlers)
+      - [Multiple Handlers with a Matching Contact](#multiple-handlers-with-a-matching-contact)
+      - [Multiple Handlers with a Single Matching Contact](#multiple-handlers-with-a-single-matching-contact)
+      - [Multiple Handlers with a Single Non-Matching Contact](#multiple-handlers-with-a-single-non-matching-contact)
+      - [Multiple Handlers with Multiple Matching Contacts](#multiple-handlers-with-multiple-matching-contacts)
+      - [Multiple Handlers with Multiple Non-Matching Contacts](#multiple-handlers-with-multiple-non-matching-contacts)
+      - [Multiple Handlers With Some Matching Contacts](#multiple-handlers-some-matching-contacts)
```

Comment:
Got a capital `W` here on with. Link is missing a `with-` after `handlers-` to work.